helm/templates/hpa-controller.yaml

```diff
@@ -26,4 +26,7 @@ spec:
         name: memory
         targetAverageUtilization: {{ .Values.controller.autoscaling.targetMemoryUtilizationPercentage }}
     {{- end }}
+    {{- with .Values.controller.autoscaling.customMetrics }}
```

Comment:
does this need to be an `OR` with the cpu/memory or is it an `AND` (what it is here)?

Reply:
it's always AND, it will choose the metric that will scale to the highest number of replicas